packages/app/src/runner/SpecRunnerDropdown.vue

```diff
@@ -61,14 +64,15 @@ import { Popover, PopoverButton, PopoverPanel } from '@headlessui/vue'
 const props = withDefaults(defineProps<{
   variant?: 'panel'
   align?: 'left' | 'right'
-
+  minimal?: boolean
```

Comment:
can we add a comment or two describing the new `minimal` option here?

Reply:
From what I can see one allows for a visual dropdown with a chevron vs the browser icon being the dropdown?